tests/contrib/test_caching.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from django.core.cache import cache
+from django.urls import reverse
+
+from adhocracy4.projects.enums import Access
+from adhocracy4.test.factories import ProjectFactory
+from meinberlin.apps.contrib import caching
+from meinberlin.test.factories.extprojects import ExternalProjectFactory
+from meinberlin.test.factories.plans import PlanFactory
+from meinberlin.test.factories.projectcontainers import ProjectContainerFactory
+
+
+def test_cache_delete():
+    ...
+
+
+def test_cache_add_or_query():
+    ...
+
+
+def test_cache_add_or_serialize():
+    ...
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "url_name,factory,factory_kwargs",
+    [
+        ("projects-list", ProjectFactory, {}),
+        ("plans-list", PlanFactory, {}),
+        ("extprojects-list", ExternalProjectFactory, {"access": Access.PUBLIC}),
+        ("containers-list", ProjectContainerFactory, {}),
+        # ("projects-list", ProjectFactory, {"access": Access.PRIVATE}),
+    ],
+)
+def test_list_url(client, url_name, factory, factory_kwargs):
+    n_objects = 3
+    url = reverse(url_name)
+    cache_key = caching.get_key(namespace=url)
```

Comment:
oops, now i realize it, the use of the URL includes slashes, and not only the viewset's basename. let's just use hardcoded values.

Reply:
done